### PR TITLE
Add feign client with `startEvent` method

### PR DESCRIPTION
**Pull request by rkondratowicz**
matches this from original ccd-client:
https://github.com/hmcts/ccd-client/blob/master/src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java#L48

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,11 +19,15 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.google.guava', name: 'guava', version: '11.0.2'
+    implementation group: 'com.google.guava', name: 'guava', version: '11.0.2'
```
Comment (doncem) [inline build.gradle:22]: only now seeing this: `11.0.2` 😳. pretty old

File: build.gradle [hunk 2]
```diff
@@ -19,11 +19,15 @@ repositories {
...
+    implementation group: 'io.github.openfeign', name: 'feign-core', version: '10.2.3'
```
Comment (doncem) [inline build.gradle]: `feign-core` is direct dependency of `feign-jackson`. i believe it'll get automatically imported as dependency 😉

**Review by doncem — APPROVED** [review]

File: build.gradle (hunk 2 above)
Comment (rkondratowicz) [reply build.gradle]: fixed

File: build.gradle (hunk 1 above)
Comment (rkondratowicz) [reply build.gradle:22]: good point. gradle file needs some cleanup in general, I'll do that in next PR

**Pull request merged**